core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1516,7 +1516,15 @@ table.tl_listing {
 	background: var(--table-header);
 }
 
-.tl_right button, .tl_content_right button {
+.tl_right > nav > ul,
+.tl_right_nowrap > nav > ul,
+.tl_content_right > nav > ul {
+	display: flex;
+	justify-content: flex-end;
+	gap: 3px;
+}
```

Comment:
This might pose some problems such as icon alignment within the new nav / ul / li structure.

See files for example:
![image](https://github.com/user-attachments/assets/923211b0-714c-4a1a-ac4d-41bb6bfed5bf)

could be fixed by omitting the `display: flex` completely and using something like this:
```css
:is(.tl_content_right, .tl_right, .tl_right_nowrap) > nav > ul > li {
    display: inline-block;
}

/* vertically align the context-menu button with the image actions */
.operations-menu-container > button {
    vertical-align: middle;
}
```
![image](https://github.com/user-attachments/assets/ad86af83-9319-4d2c-a57e-a3f43a9b9c02)